src/layers/event/api_step_chain/__init__.py

```diff
@@ -0,0 +1,33 @@
+from types import ModuleType
+
+from event.logging.step_decorators import logging_step_decorators
+from event.response.steps import response_steps
+from event.step_chain import StepChain
+from event.versioning.constants import VERSIONING_STEP_ARGS
+from event.versioning.steps import get_steps_for_requested_version, versioning_steps
+
+STEP_DECOS = [*logging_step_decorators]
```

Comment:
Little colloquial here? 😂

Reply:
it made the rest of a file a bit easier to read than when it forced new lines everywhere

Reply:
fine done :)